e2e/tests/interchain_accounts/gov_test.go

```diff
@@ -0,0 +1,149 @@
+		s.Require().NoError(err)
+
+		resp, err := s.BroadcastMessages(ctx, chainA, controllerAccount, msgSubmitProposal)
+		s.AssertValidTxResponse(resp)
+		s.Require().NoError(err)
+	})
+
+	s.Require().NoError(chainA.VoteOnProposalAllValidators(ctx, "1", cosmos.ProposalVoteYes))
+
+	time.Sleep(testvalues.VotingPeriod)
+	time.Sleep(5 * time.Second)
+
+	proposal, err := s.QueryProposalV1(ctx, chainA, 1)
+	s.Require().NoError(err)
+	s.Require().Equal(govtypesv1.StatusPassed, proposal.Status)
+
+	t.Run("start relayer", func(t *testing.T) {
+		s.StartRelayer(relayer)
+	})
+
+	s.Require().NoError(test.WaitForBlocks(ctx, 10, chainA, chainB))
+
+	var interchainAccAddr string
+	t.Run("verify interchain account", func(t *testing.T) {
+		var err error
+		interchainAccAddr, err = s.QueryInterchainAccount(ctx, chainA, govModuleAddress.String(), ibctesting.FirstConnectionID)
+		s.Require().NoError(err)
+		s.Require().NotZero(len(interchainAccAddr))
+
+		channels, err := relayer.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
+		s.Require().NoError(err)
+		s.Require().Equal(len(channels), 2)
+	})
+
+	t.Run("interchain account executes a bank transfer on behalf of the corresponding owner account", func(t *testing.T) {
+		t.Run("fund interchain account wallet", func(t *testing.T) {
+			// fund the host account, so it has some $$ to send
+			err := chainB.SendFunds(ctx, ibctest.FaucetAccountKeyName, ibc.WalletAmount{
+				Address: interchainAccAddr,
+				Amount:  testvalues.StartingTokenAmount,
+				Denom:   chainB.Config().Denom,
+			})
+			s.Require().NoError(err)
+		})
+
+		t.Run("create msg submit proposal", func(t *testing.T) {
+			msgBankSend := &banktypes.MsgSend{
+				FromAddress: interchainAccAddr,
+				ToAddress:   chainBAddress,
+				Amount:      sdk.NewCoins(testvalues.DefaultTransferAmount(chainB.Config().Denom)),
+			}
+
+			cdc := testsuite.Codec()
+			bz, err := icatypes.SerializeCosmosTx(cdc, []proto.Message{msgBankSend})
+			s.Require().NoError(err)
+
+			packetData := icatypes.InterchainAccountPacketData{
+				Type: icatypes.EXECUTE_TX,
+				Data: bz,
+				Memo: "e2e",
+			}
+
+			msgSendTx := controllertypes.NewMsgSendTx(govModuleAddress.String(), ibctesting.FirstConnectionID, uint64(time.Hour.Nanoseconds()), packetData)
+			msgs := []sdk.Msg{msgSendTx}
+			msgSubmitProposal, err := govtypesv1.NewMsgSubmitProposal(msgs, sdk.NewCoins(sdk.NewCoin(chainA.Config().Denom, govtypesv1.DefaultMinDepositTokens)), controllerAddress, "")
+			s.Require().NoError(err)
+
+			resp, err := s.BroadcastMessages(ctx, chainA, controllerAccount, msgSubmitProposal)
+			s.AssertValidTxResponse(resp)
+			s.Require().NoError(err)
+		})
+
+		s.Require().NoError(chainA.VoteOnProposalAllValidators(ctx, "2", cosmos.ProposalVoteYes))
+
+		time.Sleep(testvalues.VotingPeriod)
+		time.Sleep(5 * time.Second)
+
+		proposal, err := s.QueryProposalV1(ctx, chainA, 2)
+		s.Require().NoError(err)
+		s.Require().Equal(govtypesv1.StatusPassed, proposal.Status)
```

Comment:
Similar to the previous comment: `t.Run("vote on proposal for MsgSendTx", ...)`?